tests/ops/fixtures/saas/shippo_fixtures.py

```diff
@@ -0,0 +1,180 @@
+    load_config_with_replacement,
+    load_dataset_with_replacement,
+)
+from fides.lib.cryptography import cryptographic_util
+from tests.ops.test_helpers.vault_client import get_secrets
+
+secrets = get_secrets("shippo")
+
+
+@pytest.fixture(scope="session")
+def shippo_secrets(saas_config):
+    return {
+        "domain": pydash.get(saas_config, "shippo.domain") or secrets["domain"],
+        "api_key": pydash.get(saas_config, "shippo.api_key") or secrets["api_key"]
+    }
+
+
+@pytest.fixture(scope="session")
+def shippo_identity_email(saas_config):
+    return (
+        pydash.get(saas_config, "shippo.identity_email") or secrets["identity_email"]
+    )
+
+
+@pytest.fixture(scope="function")
+def shippo_erasure_identity_email() -> str:
+    return f"{cryptographic_util.generate_secure_random_string(13)}@email.com"
+
+
+@pytest.fixture
+def shippo_config() -> Dict[str, Any]:
+    return load_config_with_replacement(
+        "data/saas/config/shippo_config.yml",
+        "<instance_fides_key>",
+        "shippo_instance",
+    )
+
+
+@pytest.fixture
+def shippo_dataset() -> Dict[str, Any]:
+    return load_dataset_with_replacement(
+        "data/saas/dataset/shippo_dataset.yml",
+        "<instance_fides_key>",
+        "shippo_instance",
+    )[0]
+
+
+@pytest.fixture(scope="function")
+def shippo_connection_config(
+    db: Session, shippo_config, shippo_secrets
+) -> Generator:
+    fides_key = shippo_config["fides_key"]
+    connection_config = ConnectionConfig.create(
+        db=db,
+        data={
+            "key": fides_key,
+            "name": fides_key,
+            "connection_type": ConnectionType.saas,
+            "access": AccessLevel.write,
+            "secrets": shippo_secrets,
+            "saas_config": shippo_config,
+        },
+    )
+    yield connection_config
+    connection_config.delete(db)
+
+
+@pytest.fixture
+def shippo_dataset_config(
+    db: Session,
+    shippo_connection_config: ConnectionConfig,
+    shippo_dataset: Dict[str, Any],
+) -> Generator:
+    fides_key = shippo_dataset["fides_key"]
+    shippo_connection_config.name = fides_key
+    shippo_connection_config.key = fides_key
+    shippo_connection_config.save(db=db)
+
+    ctl_dataset = CtlDataset.create_from_dataset_dict(db, shippo_dataset)
+
+    dataset = DatasetConfig.create(
+        db=db,
+        data={
+            "connection_config_id": shippo_connection_config.id,
+            "fides_key": fides_key,
+            "ctl_dataset_id": ctl_dataset.id,
+        },
+    )
+    yield dataset
+    dataset.delete(db=db)
+    ctl_dataset.delete(db=db)
+
+
+@pytest.fixture(scope="function")
+def shippo_create_erasure_data(
+    shippo_connection_config: ConnectionConfig, shippo_erasure_identity_email: str
+) -> None:
+
+    sleep(60)
+
+    shippo_secrets = shippo_connection_config.secrets
+    base_url = f"https://{shippo_secrets['domain']}"
+    headers = {
+        "Authorization": f"ShippoToken {jira_secrets['api_key']}",
```

Comment:
Looks like a copy paste error here, can you rename `jira_secrets` to `shippo_secrets`